### PR TITLE
Admin Module-List Daily Menus

### PR DESCRIPTION
**Pull request by hungbkdn**

### CODE REVIEW

File: app/DailyMenu.php [hunk 1]
```diff
@@ -6,5 +6,22 @@
 
 class DailyMenu extends Model
 {
-    //
+    /**
```
Comment (at-duyho) [inline app/DailyMenu.php]: why don't you define table name for ORM model.

File: app/DailyMenu.php [hunk 2]
```diff
@@ -6,5 +6,22 @@
...
+    public $timestamps = true;
+    
+    /**
+     * Return all menu item in database with paginate by 10 item per page
+     *
+     * @return array Menu object
+     */
+    public static function getAll()
+    {
+        return DB::table('daily_menus')
```
Comment (ghost) [inline app/DailyMenu.php]: You should use self instead of DB.

File: app/Http/Controllers/Auth/RegisterController.php [hunk 3]
```diff
@@ -65,7 +65,7 @@ protected function validator(array $data)
     protected function create(array $data)
     {
         return User::create([
```
Comment (ghost) [inline app/Http/Controllers/Auth/RegisterController.php]: you should define key of array data same with column name.

File: app/Http/Controllers/Auth/RegisterController.php [hunk 4]
```diff
@@ -65,7 +65,7 @@ protected function validator(array $data)
...
-            'name' => $data['name'],
+            'full_name' => $data['name'],
             'email' => $data['email'],
             'password' => bcrypt($data['password']),
```
Comment (ghost) [inline app/Http/Controllers/Auth/RegisterController.php]: you should use event of model. https://laravel.com/docs/5.4/eloquent#events

**Review by ghost — CHANGES_REQUESTED** [review]
check my comment. thank you!

File: app/Http/Controllers/Auth/RegisterController.php (hunk 3 above)
Comment (hungbkdn) [reply app/Http/Controllers/Auth/RegisterController.php]: From @at-dungvan code

File: app/Http/Controllers/Auth/RegisterController.php (hunk 4 above)
Comment (hungbkdn) [reply app/Http/Controllers/Auth/RegisterController.php]: From @at-dungvan code, i just edit to use register and login function

**Comment by hungbkdn** [comment]
I updated, please check it again
THank you so much :smile:

File: app/DailyMenu.php [hunk 5]
```diff
@@ -6,5 +6,27 @@
...
+    protected $table = 'daily_menus';
+
+    /**
+     * Return all menu item in database with paginate by 10 item per page
+     *
+     * @return array Menu object
+     */
+    public static function getAll()
+    {
+        return DailyMenu::select('date')
```
Comment (ghost) [inline app/DailyMenu.php]: why don't you use self?

File: app/Http/Controllers/Auth/RegisterController.php (hunk 3 above)
Comment (ghost) [reply app/Http/Controllers/Auth/RegisterController.php]: so, you have to remove this file in pull request.

**Review by ghost — CHANGES_REQUESTED** [review]
please check my comment. please.
and you have to use: git comment --amend and git push --force origin "name branch" when you edit pull request.

**Comment by hungbkdn** [comment]
I updated and i forgot doing git commit --amend. I will do it for later
Thank you so much

File: app/DailyMenu.php [hunk 6]
```diff
@@ -6,5 +6,27 @@
...
+     * @var string
+     */
+    protected $table = 'daily_menus';
+
+    /**
+     * Return all menu item in database with paginate by 10 item per page
+     *
+     * @return array Menu object
+     */
+    public static function getAll()
```
Comment (ghost) [inline app/DailyMenu.php]: remove static.

File: app/Http/Controllers/DailyMenuController.php [hunk 7]
```diff
@@ -3,8 +3,20 @@
 namespace App\Http\Controllers;
 
 use Illuminate\Http\Request;
+use App\DailyMenu;
+use App\Food;
 
 class DailyMenuController extends Controller
 {
-    //
+    /**
```
Comment (at-duyho) [inline app/Http/Controllers/DailyMenuController.php]: you should declare constructor function and use DI in laravel.

**Review by ghost — CHANGES_REQUESTED** [review]
Check my comment. please.

**Comment by hungbkdn** [comment]
I updated, please check it again!
Thank you so much

File: app/Http/Controllers/DailyMenuController.php [hunk 8]
```diff
@@ -3,8 +3,39 @@
...
+     * The Daily Menu implementation.
+     *
+     * @var DailyMenu
+     */
+    protected $dailyMenus;
```
Comment (ghost) [inline app/Http/Controllers/DailyMenuController.php]: -object must be a singular noun

File: app/Http/Controllers/DailyMenuController.php [hunk 9]
```diff
@@ -3,8 +3,39 @@
...
+
+    /**
+     * Create a new controller instance.
+     *
+     * @param DailyMenu $dailyMenus instance of DailyMenu
+     *
+     * @return void
+     */
+    public function __construct(DailyMenu $dailyMenus)
```
Comment (ghost) [inline app/Http/Controllers/DailyMenuController.php]: -object must be a singular noun

**Comment by hungbkdn** [comment]
I updated, please check it again, thank you so much

**Review by ghost — COMMENTED** [review]
check comment.

**Review by ghost — APPROVED** [review]
Check comment.

File: app/DailyMenu.php [hunk 10]
```diff
@@ -18,4 +29,17 @@ public function foods()
...
     }
+
+    /**
+     * Return all menu item in database with paginate by 10 item per page
+     *
+     * @return array Menu object
+     */
+    public function getAll()
+    {
+        return self::select('date')
```
Comment (ghost) [inline app/DailyMenu.php]: -you should move this code in controller.

File: app/Http/Controllers/DailyMenuController.php [hunk 11]
```diff
@@ -3,8 +3,39 @@
 namespace App\Http\Controllers;
 
 use Illuminate\Http\Request;
+use App\DailyMenu;
+use App\Food;
```
Comment (ghost) [inline app/Http/Controllers/DailyMenuController.php]: Do you use Food?

**Review by ghost — CHANGES_REQUESTED** [review]
Check comment

**Review by ghost — APPROVED** [review]
LGTM

**Pull request merged**